client/src/components/pages/MembersPage/components/MemberCard.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import styled from 'styled-components'
+
+function Member({ member, deleteMember }) {
+  return (
+    <MemberCard key={member.id + '_visualisation'}>
+      <DeleteMember
+        key={member.id + '_deleteButton'}
+        onClick={() => {
+          deleteMember(member.id.toString())
+        }}
+      >
+        <i
+          className="fas fa-minus-circle"
+          style={{ background: 'transparent' }}
+        ></i>
+      </DeleteMember>
+
+      <Avatar
+        src={member.avatar ? member.avatar : './guitars.png'}
+        key={member.id + '_avatar'}
+      />
+      <Column key={member.id + '_info'}>
+        <Name key={member.name + member.id}> {member.name}</Name>
+        <Row key={member.id + '_roles'}>
+          {member.roles &&
+            member.roles.map(role => (
+              <Tag key={member.id + '_role_' + role}>{role}</Tag>
+            ))}
+        </Row>
+        {member.band ? member.band : member.name}:{' '}
+        {member.repetitionDayTime &&
+          member.repetitionDayTime.map(repetitionTime => (
+            <Column key={repetitionTime}>{repetitionTime}</Column>
+          ))}
+        <Row key={member.id + '_instruments'}>
+          {member.instruments &&
+            member.instruments.map(instrument => (
+              <Tag key={member.id + '_' + instrument}>{instrument}</Tag>
+            ))}
+        </Row>
+      </Column>
+    </MemberCard>
+  )
+}
+export default Member
```

Comment:
Put export in front of your function definition. The filename is MemberCard but the default export is only Member.. this is confusing.